tests/manage/pv_services/pvc_clone/test_pvc_to_pvc_clone.py

```diff
@@ -0,0 +1,104 @@
+import logging
+import pytest
+
+from ocs_ci.ocs import constants
+from ocs_ci.framework.testlib import (
+    skipif_ocs_version, ManageTest, tier1
+)
+from ocs_ci.ocs.resources import pod
+from tests import helpers
+
+logger = logging.getLogger(__name__)
+
+
+@tier1
+@skipif_ocs_version('<4.6')
+@pytest.mark.parametrize(
+    argnames=["interface_type"],
+    argvalues=[
+        pytest.param(
+            constants.CEPHBLOCKPOOL, marks=pytest.mark.polarion_id("")
+        )
+    ]
+)
+class TestPvcToPvcClone(ManageTest):
```

Comment:
@Yuggupta27 ideally this name can be TestClone.. No need for explicit mention of PVC to PVC

Reply:
Okay, I will change it  Thanks.